my-app/src/components/VideoCard/ViCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import "./ViCard.css"
+
+const ViCard = ({video}) => {
+    const {_id,title,category} = video;
+  return (
+    <div className="parent">
+        <div className="card">
+            <div className="image">
+                <img src={`https://i.ytimg.com/vi/${_id}/hqdefault.jpg`} alt="carosol"/>
```

Comment:
alt text could be something relevant like title.replaceAll(' ','-') 
this replaces all spaces and replaces with dash or you could also put underscore. SEO will be stonks.

Reply:
sure will look into this